data/json/items/migration.json

```diff
@@ -1312,6 +1317,5 @@
   {
     "id": "rifle_3006",
     "type": "MIGRATION",
-    "replace": "remington_700"
```

Comment:
You accidentally removed a line when merging.

Reply:
thanks for notifying me